src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/AssemblyNameInfoFuzzer.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Reflection.Metadata;
+using System.Runtime.InteropServices;
+
+namespace DotnetFuzzing.Fuzzers
+{
+    internal sealed class AssemblyNameInfoFuzzer : IFuzzer
+    {
+        public string[] TargetAssemblies => ["System.Reflection.Metadata"];
+
+        public string[] TargetCoreLibPrefixes => [];
+
+        public void FuzzTarget(ReadOnlySpan<byte> bytes)
+        {
+            ReadOnlySpan<char> chars = MemoryMarshal.Cast<byte, char>(bytes);
+
+            using PooledBoundedMemory<char> inputPoisonedBefore = PooledBoundedMemory<char>.Rent(chars, PoisonPagePlacement.Before);
+            using PooledBoundedMemory<char> inputPoisonedAfter = PooledBoundedMemory<char>.Rent(chars, PoisonPagePlacement.After);
+            using PooledBoundedMemory<char> inputPoisonedBoth = PooledBoundedMemory<char>.Rent(chars, PoisonPagePlacement.Before | PoisonPagePlacement.After);
```

Comment:
> `PoisonPagePlacement.Before | PoisonPagePlacement.After`

Don't think it would work as intended, it's not flag enum

Reply:
You are right:

https://github.com/dotnet/runtime/blob/47541bcece3456be3f58b15335aedbc4140b1613/src/libraries/Fuzzing/DotnetFuzzing/PooledBoundedMemory.cs#L45-L48

Will fix it, thanks!